numpy/random/src/legacy/legacy-distributions.c

```diff
@@ -84,7 +84,7 @@ double legacy_gamma(aug_bitgen_t *aug_state, double shape, double scale) {
 }
 
 double legacy_pareto(aug_bitgen_t *aug_state, double a) {
-  return exp(legacy_standard_exponential(aug_state) / a) - 1;
+  return expm1(legacy_standard_exponential(aug_state) / a);
```

Comment:
As discussed, this one ought to remain unchanged:
```suggestion
  return exp(legacy_standard_exponential(aug_state) / a) - 1;
```